peekingduck/pipeline/nodes/output/utils/csvlogger.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2021 AI Singapore
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Utils for CSV logging
+"""
+import csv
+from datetime import datetime
+from typing import Any, Dict, List
+
+class CSVLogger:
+
+    def __init__(self,filepath: str,headers: List[str],logging_interval: int=1) -> None:
+        self.headers = headers.copy()
+        self.headers.insert(0,"Time")
+        self.filepath = filepath
+        self.logging_interval = logging_interval
+        self.csv_file = open(self.filepath, mode="a+",newline='')
+        self.writer = csv.DictWriter(self.csv_file, fieldnames=self.headers)
+        self.last_write = datetime.now()
+
+    def write(self,data_pool: Dict[str, Any],specific_data: List[str]) -> None:
```

Comment:
add docstring for public methods. It will follow the format:
```
short explanation...

inputs:
outputs:
```

[fps.py](https://github.com/ongzizhao/PeekingDuck/blob/274-CSV-Writer/peekingduck/pipeline/nodes/draw/fps.py) can be a good reference.

Reply:
I have added the appropriate doc string for the public method